src/modules/visitor-space/components/VisitorSpaceSearchPage/VisitorSpaceSearchPage.tsx

```diff
@@ -631,7 +631,13 @@ const VisitorSpaceSearchPage: FC = () => {
 			return;
 		}
 		return (
-			<div className="p-visitor-space__temp-access-container">
+			<div
+				className={
+					viewMode === 'grid'
+						? 'p-visitor-space__temp-access-container--grid'
```

Comment:
Als je hier beide classes voorziet, kan je in je styling dubbele code vermijden. 
`p-visitor-space__temp-access-container p-visitor-space__temp-access-container--grid`